### ISSUE TITLE
Implement ssw for sub-domain install of wordpress multisite

### ISSUE THREAD

**Issue by neelakansha85** [neelakansha85/site-setup-wizard#41]
Feature request by a user for allowing site creation in subdomain installation using Site Setup Wizard.

**Comment by dmitriafrikanov** [comment]
I would be very grateful if you do sub domains.
Can translate and add language to Russian.

**Comment by renmarketing** [comment]
Yes, this is a great plugin, but could be better by allowing to create websites in a sub domain wordpress multisite installation.
